x-pack/plugins/reporting/public/components/screen_capture_panel_content.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// TODO: Remove once typescript definitions are in EUI
```

Comment:
I think you can remove this comment, there is no declaration here, so I think these defintions are typed in eui